tf_agents/agents/ddpg/actor_network_test.py

```diff
@@ -148,5 +149,61 @@ def testDictOfSingleAction(self):
                         [batch_size] + action_spec['motor'].shape.as_list())
     self.assertEqual(len(actor_net.trainable_variables), 2)
 
+  @test_util.run_in_graph_and_eager_modes()
+  def testHandlePreprocessingLayers(self):
+    observation_spec = (tensor_spec.TensorSpec([1], tf.float32),
+                        tensor_spec.TensorSpec([], tf.float32))
+    time_step_spec = ts.time_step_spec(observation_spec)
+    time_step = tensor_spec.sample_spec_nest(time_step_spec, outer_dims=(3,))
+
+    action_spec = tensor_spec.BoundedTensorSpec((2,), tf.float32, 2, 3)
+
+    preprocessing_layers = (tf.keras.layers.Dense(4),
+                            tf.keras.Sequential([
```

Comment:
use TF-Agents Sequential layer instead, it handles serialization better.

Reply:
replaced